### PR TITLE
work around removed procs in nim-1.1.1

### PR DESCRIPTION
**Pull request by disruptek**
This shows what needs to be done; feel free to close it in favor of your own solution. :smile:

### CODE REVIEW

File: src/terminaltables.nim [hunk 1]
```diff
@@ -200,7 +200,7 @@ proc printTable*(this: ref TerminalTable) =
 when not defined(js):
   import os, osproc
   proc termColumns(): int =
-    if os.existsEnv("COLUMNS") and getEnv("COLUMNS").isDigit():
+    if os.existsEnv("COLUMNS") and getEnv("COLUMNS")[0].isDigit():
```
Comment (xmonader) [inline src/terminaltables.nim:203]: in the latest code on master https://github.com/nim-lang/Nim/blob/master/lib/pure/includes/osenv.nim#L97 it seems to return normal tainted string? I'm not sure if get what you mean by changes in 1.1. Would you be so kind and send me the changes in Nim repo?

Comment (disruptek) [reply src/terminaltables.nim:203]: https://github.com/nim-lang/Nim/commit/3ba3307d61319984e3e8b1061f38402d53cf8f60#diff-d48a4a282bdbb4254c40b51e199e0a2cL3007

`isDigit` no longer operates upon strings, only characters...

Comment (xmonader) [reply src/terminaltables.nim:203]: ah I see now. Thanks! maybe it's better not to depend on isDigit at all then so it doesn't break on older nim versions?

Comment (disruptek) [reply src/terminaltables.nim:203]: Well, it works fine on older versions.  I would probably just try to `parseUInt` and if that raises an exception, move on...

Comment (xmonader) [reply src/terminaltables.nim:203]: Alright I'm good with that, I was worried isDigit doesn't work on chars but if that's the case I'll merge ^_^ Thank you for your contribution

**Pull request merged**